docs/cpp/c-cpp-properties-schema-reference.md

```diff
@@ -0,0 +1,126 @@
+---
+Order: 11
+Area: cpp
+TOCTitle: c_cpp_properties.json reference
+ContentId: EC1BA944-09B5-41EA-AAED-779A02C90C98
+PageTitle: c_cpp_properties.json reference
+DateApproved: 06/25/2019
+MetaDescription: Schema reference for C++ project settings in Visual Studio Code.
+---
+# c_cpp_properties.json Reference Guide
```

Comment:
Please use the update descriptions in this file `c_cpp_properties.json.md` from this change set https://github.com/microsoft/vscode-cpptools/pull/3872/files